extensions/amp-jwplayer/0.1/test/validator-amp-jwplayer.out

```diff
@@ -20,40 +20,58 @@ FAIL
-|  </head>
-|  <body>
-|    <!-- Example of a valid amp-jwplayer. -->
-|    <amp-jwplayer
-|        data-player-id="aBcD1234"
-|        data-media-id="5678WxYz"
-|        layout="responsive"
-|        width="160" height="90">
-|    </amp-jwplayer>
-|    <!-- Alternate valid example using a playlist rather than video. -->
-|    <amp-jwplayer
-|        data-player-id="aBcD1234"
-|        data-playlist-id="5678WxYz"
-|        width="160" height="90">
-|    <!-- Invalid example missing a player id. -->
-|    <amp-jwplayer
->>   ^~~~~~~~~
-amp-jwplayer/0.1/test/validator-amp-jwplayer.html:44:2 The tag 'amp-jwplayer' is missing a mandatory attribute - pick one of ['data-media-id', 'data-playlist-id']. (see https://www.ampproject.org/docs/reference/components/amp-jwplayer) [AMP_TAG_PROBLEM]
->>   ^~~~~~~~~
-amp-jwplayer/0.1/test/validator-amp-jwplayer.html:44:2 The mandatory attribute 'data-player-id' is missing in tag 'amp-jwplayer'. (see https://www.ampproject.org/docs/reference/components/amp-jwplayer) [AMP_TAG_PROBLEM]
-|        data-playerlist-id="5678WxYz"
-|        width="160" height="90">
-|    <!-- Invalid example missing playlist id or media-id. -->
-|    <amp-jwplayer
->>   ^~~~~~~~~
-amp-jwplayer/0.1/test/validator-amp-jwplayer.html:48:2 The tag 'amp-jwplayer' is missing a mandatory attribute - pick one of ['data-media-id', 'data-playlist-id']. (see https://www.ampproject.org/docs/reference/components/amp-jwplayer) [AMP_TAG_PROBLEM]
-|        data-player-id="aBcD1234"
-|        width="160" height="90">
-|  </body>
-|  </html>
+|      <head>
+|          <meta charset="utf-8">
+|          <link rel="canonical" href="./regular-html-version.html">
+|          <meta name="viewport" content="width=device-width,minimum-scale=1">
+|          <style amp-boilerplate>body{-webkit-animation:-amp-start 8s steps(1,end) 0s 1 normal both;-moz-animation:-amp-start 8s steps(1,end) 0s 1 normal both;-ms-animation:-amp-start 8s steps(1,end) 0s 1 normal both;animation:-amp-start 8s steps(1,end) 0s 1 normal both}@-webkit-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-moz-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-ms-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-o-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}</style><noscript><style amp-boilerplate>body{-webkit-animation:none;-moz-animation:none;-ms-animation:none;animation:none}</style></noscript>
+|          <noscript><style amp-boilerplate>body {-webkit-animation: none;-moz-animation: none;-ms-animation: none;animation: none}</style></noscript>
+>>         ^~~~~~~~~
+amp-jwplayer/0.1/test/validator-amp-jwplayer.html:27:8 The tag 'noscript enclosure for boilerplate' appears more than once in the document. (see https://github.com/ampproject/amphtml/blob/master/spec/amp-boilerplate.md) [MANDATORY_AMP_TAG_MISSING_OR_INCORRECT]
```

Comment:
Missed this before. Please remove the redundant `<noscript>` and rerun `--update_tests`.

Reply:
@alanorozco  Just to confirm, this was not something I had added to the `validator-amp-jwplayer.html` file before I had run `--update_tests`. Should I still proceed with removing?

Reply:
I can see it was added in the diffs for both the input and output. See the end of L26 and the beginning of L27. Looks like L27 was added.

https://github.com/ampproject/amphtml/blob/21f229e47541b504f5ec707fb86e47f9668a94b5/extensions/amp-jwplayer/0.1/test/validator-amp-jwplayer.html#L26-L27